packages/utilities/src/lib/objectEntries.ts

```diff
@@ -0,0 +1,5 @@
+import type { NonNullObject } from './utilityTypes';
+
+export function objectEntries<T extends NonNullObject>(obj: T): [keyof T, T[keyof T]][] {
+	return Object.entries(obj) as [keyof T, T[keyof T]][];
```

Comment:
same as the other 2 prs. You can better make 2 generic typings in `utility-types` files which can give all keys type and all values type. And use that in these 3 functions